scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/soic.yaml

```diff
@@ -164,6 +164,59 @@ SOIC-8-1EP_3.9x4.9mm_P1.27mm_EP2.41x3.81mm:
     # bottom_pad_size:
     paste_avoid_via: False
 
+SOIC-8-1EP_3.9x4.9mm_P1.27mm_EP2.41x3.1mm:
```

Comment:
This one has the mask size in the footprint name. Is this automatic in the script is the mask size is specified?

Reply:
See line number 222:
```yaml
  EP_mask_x: 2.71
```